system/security/plugins/htpasswdSecurity/initfiles/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2014 HPCC Systems.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Searching the HPCC-Platform tree I found this. Maybe under a different Jira we should fix them all?
  C:\HPCC\HPCC-Platform\CMakeLists.txt(50):cmake_minimum_required(VERSION 2.8.11)
  C:\HPCC\HPCC-Platform\docs\CMakeLists.txt(17):cmake_minimum_required(VERSION 2.8)
  C:\HPCC\HPCC-Platform\initfiles\CMakeLists.txt(17):cmake_minimum_required(VERSION 2.8)
  C:\HPCC\HPCC-Platform\lib2\CMakeLists.txt(17):cmake_minimum_required(VERSION 2.8)